tests/std/tests/GH_002030_asan_annotate_string/test.cpp

```diff
@@ -1,15 +1,11 @@
 // Copyright (c) Microsoft Corporation.
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception

```

Comment:
May want to also test everything works for x86 `string`s when forced on a 4-byte boundary (I think by default the compiler won't do this, but it is technically allowed) and for a custom allocator that doesn't return 8-byte aligned addresses. The tests in the PR I abandoned covered these if you want to take them.